pinot-core/src/main/java/com/linkedin/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -1002,6 +1008,7 @@ public LLRealtimeSegmentDataManager(RealtimeSegmentZKMetadata segmentZKMetadata,
       }
     }
 
+    realtimeSegmentConfigBuilder.setUpdateMetrics(indexingConfig.getUpdateMetrics());
```

Comment:
can u pls move this up to line 989, so we have the builder in one place?

Reply:
Yes, done.